core/src/main/java/tc/oc/pgm/command/FreezeCommand.java

```diff
@@ -0,0 +1,79 @@
+package tc.oc.pgm.command;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.CommandException;
+import app.ashcon.intake.parametric.annotation.Switch;
+import java.util.stream.Collectors;
+import net.kyori.text.Component;
+import net.kyori.text.TextComponent;
+import net.kyori.text.TranslatableComponent;
+import net.kyori.text.format.TextColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.modules.FreezeMatchModule;
+import tc.oc.pgm.util.chat.Audience;
+import tc.oc.pgm.util.named.NameStyle;
+
+public class FreezeCommand {
```

Comment:
I feel like freezing is a community thing?

Reply:
I have to check when I have a chance and see if it can be safely moved over, last I thought there was one PGM specific requirement needed. But I’ll get back to you on this 👍

Reply:
Was able to completely remove freeze from PGM, has been implemented in Community 😎